_includes/payment-resource.md

```diff
@@ -0,0 +1,127 @@
+| └─➔&nbsp;`method`        | string     | The HTTP method to use when performing the operation.                                                                                                                                            |
+| └─➔&nbsp;`href`          | string     | The target URI to perform the operation against.                                                                                                                                                 |
+| └─➔&nbsp;`rel`           | string     | The name of the relation the operation has to the current resource.                                                                                                                              |
+
+### Operations
+
+The operations should be performed as described in each response and not as
+described here in the documentation.
+Always use the `href` and `method` as specified in the response by finding
+the appropriate operation based on its `rel` value.
+The only thing that should be hard coded in the client is the value of
+the `rel` and the request that will be sent in the HTTP body of the request
+for the given operation.
+
+{:.table .table-striped}
+| Operation                          | Description                                                                                                                                                                                                                                                                    |
+| :--------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `update-payment-abort`        | [Aborts][abort] the payment order before any financial transactions are performed.                                                                                                                                                                               |
+| `redirect-authorization`            | Contains the URI that is used to redirect the consumer to the PayEx Payment Pages containing the card authorization UI.                                                                                                                                                                 |
+| `view-authorization`                | Contains the JavaScript href that is used to embed  the card authorization UI directly on the webshop/merchant site                                                                                                                                                                |
+
+[expansion]: /payments/credit-card/other-features/#expansion
```

Comment:
Using this link means that this include can only be used in the credit card instrument section.